src/stories/Library/Modals/modal-loan/ModalLoan.tsx

```diff
@@ -167,18 +168,11 @@ export const ModalLoan: React.FC<ModalLoanProps> = ({
   };
 
   return (
-    <div
-      className={`modal modal-loan modal-padding ${
-        shoulShowModal ? "modal-show" : ""
-      }`}
+    <ModalWrapper
```

Comment:
If you go with just Modal for the component then remember to change all these <ModalWrapper > mentions ;)